builders/toolkit/integrations/oracles/band.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using Band Protocol for Price Feed Oracles
+description: Learn how to use the Band Protocol's descentralized oracle network to get reliable token prices enabling secure data access for your Tanssi EVM appchain.
+---
+
+# Accesing Price Feeds with the Band Protocol
+
+## Introduction {: #introduction }
+
+[Band Protocol](https://www.bandprotocol.com/){target=\_blank} is a decentralized oracle network that provides reliable, secure, real-time data to smart contracts on various blockchain networks.
+
+The protocol is built on top of BandChain, an appchain designed to be compatible with most EVM-compatible chains, such as Tanssi EVM appchains, and blockchain development frameworks. The protocol aims to provide a solution that is:
+
+- Decentralized, leveraging the computational power of a network of validators
+- Flexible, supporting a wide range of data sources and formats, making integrations easy
+- Scalable, designed to handle high volumes of data requests
+- Affordable, allowing users to request data only when they need to and pay the associated fees
+
+Band protocol is currently deployed on many blockchains ([Moonbeam](https://docs.moonbeam.network/builders/integrations/oracles/band-protocol/){target=\_blank}, for example) across different ecosystems. To deploy the oracle onto your appchain, reach out to the [Band Protocol](https://www.bandprotocol.com/){target=\_blank} team directly.
+
+This tutorial will walk through the steps to interact with price feeds using the Band protocol's oracle on the [Tanssi demo EVM-compatible appchain](https://apps.tanssi.network/demo){target=\_blank}. 
+
+## Setup on the Tanssi Demo EVM Appchain {: #setup-on-demo-evm-appchain }
+
+The Band Protocol oracle is already deployed on the Tanssi demo EVM appchain and configured to provide prices for the `ETH` and `DOT` tokens.
+
+The price feeds are pushed regularly to a smart contract that is accessible at the following address:
+
+```text
+{{ networks.dancebox.oracles.band.smart_contract }}
+```
+
+The smart can be interacted with using the interface:
+
+???+ code "IStdReference.sol"
+
+    ```solidity
+    --8<-- 'code/builders/toolkit/integrations/oracles/band/IStdReference.sol'
+    ```
+
+As seen above in the interface, there are two functions for fetching data: `getReferenceData` to fetch the for a given base/quote pair, and `getReferenceDataBulk` to fetch multiple prices simultaneously.
```

Comment:
I would explain what the function returns. It returns the struct with the price, UNIX epoch of the last time when base price gets updated. and UNIX epoch of the last time when quote price gets updated.

You explain this a bit later but I think here is a bit more crucial to also explain